scout/server/blueprints/variants/forms.py

```diff
@@ -212,8 +224,6 @@ class StrFiltersForm(VariantFiltersForm):
 class SvFiltersForm(VariantFiltersForm):
     """Extends FiltersForm for structural variants"""
 
-    size = StringField("Length")
-    size_shorter = BooleanField("Length shorter than")
```

Comment:
Replacing this filter:
<img width="1144" alt="image" src="https://github.com/user-attachments/assets/90d5d698-f4d5-48ca-9a02-f5a47fc9e4cd" />

With the new one, which allows to select variants longer or shorter than a threshold